lua/orgmode/utils/fs.lua

```diff
@@ -30,11 +30,8 @@ function M.get_real_path(filepath, match)
     return false
   end
   local real = vim.loop.fs_realpath(substituted)
-  if match then
-    -- if path ends with / , real needs too
```

Comment:
Actually I find the comment useful and would not remove it.

Reply:
Sure.